WORKSPACE.in

```diff
@@ -154,19 +147,19 @@ http_archive(
 #    path = "/home/tmp/ros",
 #)
 #
-## PCL 1.9
-#new_local_repository(
-#    name = "pcl",
-#    build_file = "third_party/pcl.BUILD",
-#    path = "/usr/local/include/pcl-1.9",
-#)
-#
-#new_local_repository(
-#    name = "glew",
-#    build_file = "third_party/glew.BUILD",
-#    path = "/usr/include",
-#)
-#
+# PCL 1.11
+new_local_repository(
+    name = "pcl",
+    build_file = "third_party/pcl.BUILD",
+    path = "/usr/local/include/pcl-1.11",
+)
+
+new_local_repository(
+    name = "glew",
```

Comment:
I checked via include headers and 'deps' in BUILD files, No 'glew' dependency was found. As such, we may remove glew dependency here.

Reply:
glew is a part of OpenGL, so I guess that it is an indirectly dependency. But anyway, as long as what_works.sh works :)